vite.config.ts

```diff
@@ -38,13 +38,18 @@ export default defineConfig({
   },
   test: {
     include: [
-      '{src,test}/**\/*.{test,spec}.?(c|m)[jt]s?(x)',
+      '{src,test}/**/*.{test,spec}.?(c|m)[jt]s?(x)',
       './test/examples/*.js',
     ],
     exclude: ['dist'],
-    environmentMatchGlobs: [
-      // React example requires DOM emulation
-      ['examples/react-example/**', 'jsdom']
```

Comment:
This change of running everything in jsdom makes sense to me since it shouldn't be harmful, but I figured I'd leave a comment pointing out that it is a change regardless.